docs/conf.py

```diff
@@ -66,7 +66,7 @@
 
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
-exclude_patterns = []
+exclude_patterns = ['staging']
```

Comment:
should we add the makefiles to the excluded patterns?

Reply:
we only need to exclude rst files we don't want to include in the output and directories containing rts files to exclude or directories we don't want autodiscovery to search. Makefile is not included in the output.